CMakeLists.txt

```diff
@@ -207,7 +202,7 @@ endif()
 ### Data Components [NEMSdatm, CDEPS]
 ###############################################################################
 if(NEMSdatm)
-  add_subdirectory(DATM)
+  add_subdirectory(DATM/DATM)
```

Comment:
Why do we need this change for removing nemsio?

Reply:
`add_subdirectory(DATM)` adds both `DATM` and `tools` which depends on nemsio. ufs-weather-model depends only on `DATM/DATM`

Reply:
I agree with this change, unless the `tools` are pulled out, which they should be.  The `tools` should ideally reside in `UFS_utils`

Reply:
Or `NEMS-datm` or something like that.

Reply:
I don't believe that what is currently in the tools subdirectory is used at all. It is the original code I wrote to convert nesmio to netcdf for the DATM. 

The code which is currently used to generate the DATM files resides in the utilities subdirectory of the [DATM-MOM6-CICE6](https://github.com/NOAA-EMC/DATM-MOM6-CICE5/tree/develop/utilities) repo. That also contains the gen_scrip. It is the utilities subdirectory which would need to get moved to ufs-utils.

Reply:
@binli2337 @DeniseWorthen can we remove tools under https://github.com/noaa-emc/nemsdatm to avoid confusion?

Reply:
Yes, I believe we can & should remove the tools subdirectory.

Reply:
Yes.